pages/hub/insights/index.tsx

```diff
@@ -110,7 +248,7 @@ const InsightsHub: WithPageLayout = () => {
 };
 
 InsightsHub.PageLayout = HubLayout;
-InsightsHub.isPrivateRoute = true;
+InsightsHub.isPrivateRoute = false;
```

Comment:
refactor: are routes private by default? If not, we probably don't need to set this and can remove the line.